files/zh-tw/web/javascript/reference/global_objects/string/localecompare/index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: String.prototype.localeCompare()
+slug: Web/JavaScript/Reference/Global_Objects/String/localeCompare
+tags:
+  - Internationalization
+  - JavaScript
+  - String.prototype.localeCompare()
+  - Prototype
+  - 参考
+  - 字串
+translation_of: Web/JavaScript/Reference/Global_Objects/String/localeCompare
+---
+{{JSRef}}
+
+**`localeCompare()`** 回傳一個數字，用來表示其與被比較的字串的先後順序。
+
+如果環境中有支援[`Intl.Collator` API](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator)，這個方法實際上是調用 `Intl.Collator` API。
```

Comment:
follow https://github.com/mdn/translated-content/blob/main/docs/zh-tw/translation-guide.md#%E7%AB%99%E5%85%A7%E9%80%A3%E7%B5%90%E7%BF%BB%E8%AD%AF

```suggestion
如果環境中有支援[`Intl.Collator` API](/zh-TW/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator)，這個方法實際上是調用 `Intl.Collator` API。
```

Reply:
There are a lot of links needed to be updated, please follow this one.